sbpy/spectroscopy/core.py

```diff
@@ -9,29 +9,270 @@
 
 from abc import ABC
 import numpy as np
+import astropy.constants as con
 import astropy.units as u
-import astropy.constants as const
+from astroquery.jplspec import JPLSpec
+from astropy.time import Time
+from astroquery.jplhorizons import Horizons
+from ..activity.gas import photo_timescale
+
 
 __all__ = ['Spectrum', 'SpectralModel']
 
 
-def einstein_coeff(frequency):
+def constants(temp_estimate, transition_freq, mol_tag, vgas):
```

Comment:
This function could have a more specific name like `molecular_data`. Also make sure that it does its specific task, for example the coma expansion velocity is not neede as an argument.

Reply:
Done, will commit soon. I also changed something to make the intensity line integral easier to implement by itself, so give me a chance to commit it so you can take a look at the new version of core